### ISSUE TITLE
Safe items

### ISSUE THREAD

**Issue by AceCap** [AceCap/Bloodshed-RSPS#2]
items are being dropped in safe areas. would like to make them safe in safe areas and still have items dropped in wilderness

**Comment by dosier** [comment]
On death u mean?

**Comment by AceCap** [comment]
yes sorry, when someone tried to fix for me they made items safe in non pvp but it made items safe in wilderness aswell. then when someone fixed again it made items drop in both areas.
